internal/functions/new/templates/config.toml

```diff
@@ -2,6 +2,7 @@
 [functions.{{ . }}]
 enabled = true
 verify_jwt = true
+use_package_json = false
```

Comment:
I'm not sure if this is the right fix. Based on [their example](https://github.com/supabase/supabase/issues/38402#issuecomment-3298859419), the error seems to be caused by [health function](https://github.com/Johnny5AtGithub/supabase-todos-demo/blob/master/supabase/functions/health/index.ts#L4) failing to import package.json on Windows.

Most likely we have a bug in path resolution that's preventing package.json at root repo directory from being volume mounted into the edge runtime container.